packages/contracts/as/NFT/assembly/index.ts

```diff
@@ -409,6 +502,14 @@ export function init(
         )
     )
 
+    // find storage usage of a single account
+    const account_id = 'a'.repeat(64)
+    const initial_storage: u128 = u128.from(context.storageUsage)
+    accounts.set(account_id, u128.from('0'))
```

Comment:
As I commented in `models.ts` you are calculating the storage costs of storing users' amount of tokens while here we store `NFTs`.